<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>liuyanghejerry@126.com</author>
 <class>ColorInspector</class>
 <widget class="QWidget" name="ColorInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>122</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QRadioButton" name="hsv">
       <property name="text">
        <string>HSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rgb">
       <property name="text">
        <string>RGB</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="red">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="RedspinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="green">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="GreenspinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="blue">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="BluespinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="alpha">
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="AlphaspinBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="3">
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hsv</tabstop>
  <tabstop>rgb</tabstop>
  <tabstop>RedspinBox</tabstop>
  <tabstop>GreenspinBox</tabstop>
  <tabstop>BluespinBox</tabstop>
  <tabstop>AlphaspinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
